src/Qir/Runtime/lib/QSharpCore/qsharp__core__qis.hpp

```diff
@@ -7,6 +7,20 @@
 
 struct QirArray;
 
+struct QirRTuple
+{
+    PauliId pauli;
+    double angle;
+    QUBIT* target;
+};
+
+struct QirExpTuple // NOLINT
+{
+    QirArray* paulis;
+    double angle;
+    QirArray* targets;
+};
```

Comment:
Curious: Are you sure the name `QirExpTuple` requires `// NOLINT`? (if yes then I'm surprised why it does, but the names  `QirRTuple` and `QirArray` do not require `// NOLINT`)

Reply:
nope, this is definitely an oversight leftover from a previous name I was experimenting with. You are right that it can just be removed.